sinks/api/http/endpoint.go

```diff
@@ -13,8 +13,27 @@ import (
 	"github.com/go-kit/kit/endpoint"
 	"github.com/ns1labs/orb/pkg/types"
 	"github.com/ns1labs/orb/sinks"
+	"github.com/ns1labs/orb/sinks/backend"
+	"strings"
 )
 
+var restrictiveKeyPrefixes = []string{backend.ConfigFeatureTypePassword}
+
+func omitSecretInformation(metadata types.Metadata) (restrictedMetadata types.Metadata) {
+	metadata.RestrictKeys(func(key string) bool {
+		match := false
+		for _, restrictiveKey := range restrictiveKeyPrefixes {
+			if match {
```

Comment:
since we are matching on explicit keys now, there was no reason to do contains or lower... just a strait match